middlewares/validators/userStatus.js

```diff
@@ -54,9 +54,21 @@ const validateUserStatusData = async (todaysTime, req, res, next) => {
       committed: Joi.number().required(),
       updatedAt: Joi.number().required(),
     }),
-  }).or("currentStatus", "monthlyHours");
+  });
 
+  const cancelOooSchema = Joi.object()
+    .keys({
+      cancelOoo: Joi.boolean().valid(true).required(),
+    })
+    .unknown(false);
+
+  let schema;
   try {
+    if (Object.keys(req.body).includes("cancelOoo")) {
```

Comment:
Maybe we should create a constant for `cancelOoo`. This way we can make it consistent everywhere `cancelOoo` is used.

Reply:
Done